scripts/release

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+const execSync = require('child_process').execSync;
+const minimist = require('minimist');
+const standardVersion = require('standard-version');
+const chalk = require('chalk');
+const args = minimist(process.argv.slice(2));
+
+const DEFAULT_UPSTREAM_REPO_ORIGIN = 'origin';
+const DEFAULT_UPSTREAM_REPO_BRANCH = 'master';
+const origin = args['repo-origin'] || DEFAULT_UPSTREAM_REPO_ORIGIN;
+const branch = args['repo-branch'] || DEFAULT_UPSTREAM_REPO_BRANCH;
+
+console.log(chalk.blue('ℹ'), `Fetching from origin '${origin}' to read upstream version...`);
+try {
+  console.log(execSync(`git fetch ${origin}`).toString())
+} catch (e) {
+  console.log(chalk.red('✘'), 'Couldn\'t fetch latest commits. Please check the error above.');
+  return;
+}
+
+const localRef = execSync(`git rev-parse ${branch}`).toString();
+const originRef = execSync(`git rev-parse ${origin}/${branch}`).toString();
+
+if (localRef !== originRef) {
+  console.log(chalk.red('✘'), `Local branch '${branch}' is not up to date with '${origin}/${branch}'. Please update your local branch first.`);
+  return;
+}
+
+console.log(chalk.green('✔'), 'Release branch is up to date with remote branch.');
+
+standardVersion({
+  dryRun: args['dry-run'],
+  prerelease: args.prerelease,
+  sign: args.sign,
+  releaseAs: args['release-as']
+}).then(() => {
+  if (!args['dry-run']) {
+    console.log(chalk.blue('ℹ'), `Pushing release commit to origin '${origin}' on branch '${branch}'...`);
+    try {
+      const output = execSync(`git push ${origin} ${branch} --follow-tags`).toString();
+      console.log(chalk.green(output));
+      console.log(chalk.green('✔'), 'Successfully pushed release commit');
+    } catch (e) {
+      console.log(chalk.red('✘'), 'Couldn\'t push release commit. Please check the error above.');
+      return { error: true };
+    }
+  } else {
+    console.log(chalk.blue('ℹ'), 'This is a dry run. Nothing\'s being pushed.');
+  }
+}).then(error => {
+  if (error) {
+    return error;
+  }
+  if (!args['dry-run']) {
+    console.log(chalk.blue('ℹ'), 'Publishing new Embark version on npm...');
+
+    const npmPublishCommand = args.prerelease ? 'npm publish --tag prerelease' : 'npm publish';
```

Comment:
Re: `--tag prerelease`, should `prerelease` be a variable determined by another command-line option?

For example, we may be doing an `alpha` prelease and bumping from `alpha.22` to `alpha.23` while wanting to `--tag` it on the npm registry as `next`.